### ISSUE TITLE
LearningWithNoisyLabels fit - ValueError: Input contains NaN, infinity or a value too large for dtype('float64').

### ISSUE THREAD

**Issue by ke-sun-ds** [cleanlab/cleanlab#91]
```
from cleanlab.classification import LearningWithNoisyLabels
from sklearn.linear_model import LogisticRegression

# Wrap around any classifier (scikit-learn, PyTorch, TensorFlow, FastText, etc.)
lnl = LearningWithNoisyLabels(clf=LogisticRegression()) 
lnl.fit(X=X_train_data.to_numpy(), s=train_noisy_labels.to_numpy()) 
```
I've double checked there is **NO** NaN in X_train_data.to_numpy() and train_noisy_labels.to_numpy(), but when fit model it throws `ValueError: Input contains NaN, infinity or a value too long for dtype('float')` 

X_train_data is a (10957, 297) pd.DataFrame, train_noisy_labels is a (10957,) pd.Series without any NaN

any idea why could this happen?

**Comment by anishathalye** [comment]
Cross-ref: #51

**Comment by anishathalye** [comment]
Fixed by #149.
